test/standalone/test-capture-time.js

```diff
@@ -0,0 +1,109 @@
+// /**
```

Comment:
So now that I have uncommented and run the tests I realized why I commented it out in the first place - `test-capture-time.js` frequently failed (as far as I have seen) because it has fairly strict requirements on the capture time (<20ms) that can't be met. I was wondering, should I relax this requirement?